util/httputil/cors.go

```diff
@@ -20,14 +20,14 @@ import (
 )
 
 var corsHeaders = map[string]string{
-	"Access-Control-Allow-Headers":  "Accept, Authorization, Content-Type, Origin",
-	"Access-Control-Allow-Methods":  "GET, POST, OPTIONS",
```

Comment:
I don't think removing `OPTIONS` in here is related to the change.
IIUC, this is not needed as Prometheus doesn't offer actual OPTIONS endpoints and I don't think CORS spec would require this chicken-and-egg setting for preflights to work.
So I agree it's not needed, but maybe let's do it in another PR instead of hiding it in here so those who think this may break something can speak out.

Reply:
Fair. This can be done in a separate PR.